src/features/course-main/course-main.tsx

```diff
@@ -25,7 +28,11 @@ export const CourseMain = ({ courseName, type }: CourseMainProps) => {
   const requiredDate = new Date(course.startDate).setHours(0, 0, 0, 0);
```

Comment:
I suggest to add some work-with-date library to make the code more readable

Reply:
Added date library Day.js